api/routes/auth.py

```diff
@@ -61,13 +61,13 @@ def google_auth(auth_data: s.GoogleAuthIn, db: Session = Depends(get_db)):
         if id_info.iss not in ISSUER_WHITELIST:
             raise ValueError("Wrong issuer.")
 
-        user = db.scalar(sa.select(m.User).where(sa.and_(m.User.email == id_info.email)))
+        user = db.scalar(sa.select(m.AuthAccount).where(sa.and_(m.AuthAccount.email == id_info.email)))
```

Comment:
you also need to add an identifier (from google or apple, or other social networks in the future), together the email, type and identifier must be unique